pyproject.toml

```diff
@@ -1,2 +1,52 @@
+#https://hatch.pypa.io/latest/config/build/
+
+[project]
+name = "tutor-indigo"
+dynamic = ["version"] # hatch_build.py will set it later
```

Comment:
please move this below so that it is easy to see which items are set dynamic. See https://github.com/overhangio/tutor-minio/pull/57/files for ref